Fastfile

```diff
@@ -140,10 +134,34 @@ def _build_with_gym()
   end
 end
 
+def _get_export_method()
+  export_method = "enterprise"  # Default export method is `enterprise` since this is the most commonly used
+  if ENV['GYM_EXPORT_OPTIONS'] != nil
```

Comment:
It might be more idiomatic to use `.present?` instead of `!= nil`

Reply:
Can also do `.nil?` as well?

Reply:
Yup.